Annotation/Commentable.php

```diff
@@ -0,0 +1,14 @@
+<?php
+namespace FOS\CommentBundle\Annotation;
+
+use Doctrine\Common\Annotations\Annotation;
+
+/**
+ * @Annotation
+ * @Target("PROPERTY")
+ */
+class Commentable
+{
+    public $fetch = 'EAGER';
+    public $identifierProperty = 'entityIdentifier';
```

Comment:
please add the phpdoc on this (``@var string``) so that the annotation parser enforces it people using the annotation. and for the ``fetch`` property, it may make sense to use a ``Enum`` type